### ISSUE TITLE
Invalid definition for service "Auth0\Symfony\Controllers\AuthenticationController":

### ISSUE THREAD

**Issue by mkldevops** [auth0/symfony#189]
### Checklist

- [X] I have looked into the [Readme](https://github.com/auth0/symfony#readme) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/symfony/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

When I run following command :
```sh
APP_ENV=dev php console lint:container 
```

I have this error:
```sh                                                                                                                        
 [ERROR] Invalid definition for service "Auth0\Symfony\Controllers\AuthenticationController":                           
         "Auth0\Symfony\Controllers\AuthenticationController::__construct()" requires 3 arguments, 2 passed.            
```

### Reproduction

1. composer req auth0/symfony
2. add on bundle on `bundles.php` file
3. run the commade `php bin/console lint:container`

### jwt-auth-bundle version

5.2.3

### Symfony version

6.4

### PHP version

8.3

**Comment by abunch** [comment]
@evansims any chance you can approve/merge this PR?

**Comment by edent** [comment]
Is there a way I can manually apply this patch to my installation?

**Comment by mkilmanas** [comment]
@edent you can try switching over to [`elvi/auth0-symfony`](https://packagist.org/packages/elvi/auth0-symfony) for now, which is the fork of this repository but with the open PRs merged in. Let's hope this official bundle catches up with the world eventually and we can all come back to use it again.

**Comment by evansims** [comment]
The fix has shipped as part of [5.3.0](https://github.com/auth0/symfony/releases/tag/5.3.0); thanks for your work on this, as well as your other contributions, @mkilmanas. Much appreciated!
